### ISSUE TITLE
can't find the dbcan_utils

### ISSUE THREAD

**Issue by WeiweiguoGuo** [linnabrown/run_dbcan#159]
Hello,

when i use the "dbcan_utils", it showed "dbcan_utils: command not found...".  How can I solve this problem? I did not find "dbcan_utils" in the package.

Thank you!

**Comment by linnabrown** [comment]
dbcan_utils is not a command

**Comment by WeiweiguoGuo** [comment]
dbcan_utils fam_abund -bt Wet2014.depth.txt -i ../Wet2014.dbCAN -a TPM

What does this mean？

**Comment by linnabrown** [comment]
Oh wait a second. This is included by my labmates. Are you trying the run from raw reads procedure? If so, dbcan_utils should be installed with dbcan together.

Here is my printout for my server

```
[xxx run_dbcan]$ dbcan_utils -h
usage: 
    dbcan_utils [positional arguments] [options]
    -----------------------------------------
    fam_abund [calculate the abundance of CAZyme]. Example usage: dbcan_utils fam_abund -bt samfiles/fefifo_8002_7.depth.txt -i fefifo_8002_7.dbCAN
    -----------------------------------------
    fam_substrate_abund [calculate the abundance of CAZyme substrate]. Example usage: dbcan_utils fam_substrate_abund -bt samfiles/fefifo_8002_7.depth.txt -i fefifo_8002_7.dbCAN
    -----------------------------------------
    CGC_abund [calculate the abundance of PUL]. Example usage: dbcan_utils CGC_abund -bt samfiles/fefifo_8002_7.depth.txt -i fefifo_8002_7.dbCAN
    -----------------------------------------
    CGC_substrate_abund [calculate the abundance of PUL substrate]. Example usage: dbcan_utils CGC_substrate_abund -bt samfiles/fefifo_8002_7.depth.txt -i fefifo_8002_7.dbCAN
    

Calculate the abundance CAZyme, PUL and substrate.

positional arguments:
  {fam_abund,fam_substrate_abund,CGC_abund,CGC_substrate_abund,generate_PULfaa}
                        What function will be used to analyze?

optional arguments:
  -h, --help            show this help message and exit
  -i INPUT, --input INPUT
                        dbCAN CAZyme annotation output folder.
  -bt BEDTOOLS, --bedtools BEDTOOLS
                        bedtools gene reads count results.
  -1 R1, --R1 R1        R1 reads, support gz compress file
  -2 R2, --R2 R2        R2 reads, support gz compress file, None for single end sequencing
  -o OUTPUT, --output OUTPUT
                        output files
  -a {FPKM,RPM,TPM}, --abundance {FPKM,RPM,TPM}
                        normalized method
  --db_dir DB_DIR       dbCAN database directory
```

I think you can install this package using step by step version. dbcan_utils should be installed correctly.

Currently, we also fixed the bug in updated 4.1.4 version. However, Bioconda has some CICD issue so we could not directly update dbcan to 4.1.4. One strategy you can use is that installing via source code

1. wget https://github.com/linnabrown/run_dbcan/releases/download/4.1.4/dbcan-4.1.4.tar.gz
2. pip install dbcan-4.1.4.tar.gz

Than you can update the package.

Thank you very much for using our tool. If you have more question please let me know.
